Resources/Prototypes/SimpleStation14/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -39,7 +39,7 @@
   name: pv3 mini
   parent: BaseWeaponBatterySmall
   id: WeaponLaserPv3Mini
-  description: Standard issue Nyanotrasen laser sidearm. This thing can bullseye a two metre target, it's not impossible!
+  description: Standard issue Nanotrasen laser sidearm. This thing can bullseye a two metre target, it's not impossible!
```

Comment:
```suggestion
  description: Standard issue NanoTrasen laser sidearm. This thing can bullseye a two-meter target, it's not impossible!
```

Reply:
again with the NanoTrasen vs Nanotrasen from the previous comment along with two metre being a correct spelling in british english just not american english.

Reply:
🇺🇸

Reply:
> 🇺🇸

the SimpleStation14 location is canada

Reply:
I believe most things use EN_US.

Reply:
this spelling was already approved when it was introduced by PSP I believe its fine.

Reply:
Nanotrasen is fine, but the US/UK English should be consistent- Meter